status/flowcells.py

```diff
@@ -289,4 +289,16 @@ def post(self, flowcell):
                 self.set_header("Content-type", "application/json")
                 self.finish(json.dumps(links))
 
+#Functions
+def get_container_from_id(flowcell):
+    if flowcell[7:].startswith('00000000'):
```

Comment:
Well, tbh, I could automate the check by adding lims calls. I'd rather not
do it since lims api calls, as you know, are not the fastest thing we can
do, but it's the safest.

On Mon, Apr 27, 2015 at 1:11 PM, senthil10 <notifications@github.com> wrote:

> In status/flowcells.py
> <https://github.com/SciLifeLab/genomics-status/pull/197#discussion_r29137859>
> :
>
> > @@ -289,4 +289,16 @@ def post(self, flowcell):
> >                  self.set_header("Content-type", "application/json")
> >                  self.finish(json.dumps(links))
> >
> > +#Functions
> > +def get_container_from_id(flowcell):
> > +    if flowcell[7:].startswith('00000000'):
>
> hej, yes like @guillermo-carrasco <https://github.com/guillermo-carrasco>
> said string 00000000 will always be on MiSeq run, so guess this will
> work. But I am always worried about the number of zeros though, maybe its
> always the same but I am still paranoid [image: :sweat_smile:] .
>
> So if its *me* I will check for - in flowcell name i.e. if '-' in
> flowcell[7:] or if '-' in flowcell.split('_')[-1]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/genomics-status/pull/197/files#r29137859>.
>